concepts/05 Widgets/DateBox/00 Getting Started with DateBox/00 Getting Started with DateBox.md

```diff
@@ -0,0 +1,332 @@
+#include tutorials-intro-installationnote
+
+**DateBox** is a widget that allows users to set a certain date, time, or their combination.
+
+This tutorial shows how to configure basic **DateBox** features. The created widget allows setting the date and time from the allowed date range, logs the set value to console, and prevents users from specifying the US bank holidays and weekend days.
+
+<div class="simulator-desktop-container" data-view="/Content/Applications/20_1/GettingStartedWith/DateBox/index.html, /Content/Applications/20_1/GettingStartedWith/DateBox/index.js, /Content/Applications/20_1/GettingStartedWith/DateBox/index.css"></div>
+
+
+Refer to the subtopics for details on every configuration step. You can also see the full code below or download it from <a href="https://github.com/DevExpress-Examples/getting-started-with-datebox" target="_blank">this GitHub repository</a>:
```

Comment:
```suggestion
Refer to the subtopics for details on each configuration step. You can also see the full code below or download it from the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-datebox" target="_blank">getting-started-with-datebox</a>.
```

Reply:
"Refer to the subtopics for details on each configuration step."

Where are the links to these subtopics located?

Reply:
The links are in the left panel:
![image](https://user-images.githubusercontent.com/66619750/87015864-a1352c00-c1d6-11ea-89db-47b4054b810a.png)

Reply:
Thank you for the clarification.